src/Plateau.php

```diff
@@ -1,137 +1,92 @@
 <?php
- echo "<style>
-    .geo{
-        width:50px;
-        height:50px;
-        border: solid 1px black;
-        background-color : cyan;
-    }
-    .hist{
-        width:50px;
-        height:50px;
-        border: solid 1px black;
-        background-color : yellow;
-    }
-    .diver{
-        width:50px;
-        height:50px;
-        border: solid 1px black;
-        background-color : pink;
-    }
-    .sport{
-        width:50px;
-        height:50px;
-        border: solid 1px black;
-        background-color : orange;
-    }
-    .info{
-        width:50px;
-        height:50px;
-        border: solid 1px black;
-        background-color : green;
-    }
-    .perso{
-        width:50px;
-        height:50px;
-        border: solid 1px black;
-        background-color : purple;
-    }
-    .depart{
-        width:100px;
-        height:100px;
-        border: solid 1px black;
-        background-color : black;
-    }
-    table{
-        background:grey;
-    }
- </style>";
+namespace trivial;
 
- $taille_tab=13;
-$tableau = array();
+ class Plateau {
+    private $tableau = array();
+    private $taille_tab=13;
 
-for($i=1;$i<=$taille_tab;$i++){
-    for($j=1;$j<=$taille_tab;$j++){
-        if($i-0.5==$taille_tab/2 && $j-0.5==$taille_tab/2 ){
-            $tableau[$i][$j]="depart";
-        }
-        else
-        if($i==1 || ($i-0.5==$taille_tab/2 && $j<$taille_tab/2)){
+    public function creaPlateau(){
+    for($i=1;$i<=$this->taille_tab;$i++){
+        for($j=1;$j<=$this->taille_tab;$j++){
+            if($i-0.5==$this->taille_tab/2 && $j-0.5==$this->taille_tab/2 ){
+                $this->tableau[$i][$j]=["theme"=>"depart","player"=>array()];
+            }
+            else
+            if($i==1 || ($i-0.5==$this->taille_tab/2 && $j<$this->taille_tab/2)){
+                switch($j%6){
+                    case 0:$this->tableau[$i][$j]=["theme"=>"geo","player"=>array()];
```

Comment:
`"geo", "diver", ...` are called magic strings and are hell for maintainability. Use constants instead?